test/runner/defs.bzl

```diff
@@ -184,6 +212,8 @@ def syscall_test(
         size = size,
         platform = "native",
         use_tmpfs = False,
+        use_image = "",
+        setup_command = "",
```

Comment:
Why don't we add default arguments to `_syscall_test` then we don't have to add these 2 variables everywhere?